src/ControlSystem/Tags.hpp

```diff
@@ -184,4 +170,66 @@ struct MeasurementTimescales : db::SimpleTag {
   }
 };
 }  // namespace Tags
+
+/// \ingroup ControlSystemGroup
+/// Place to hold all options for a single control system
+namespace OptionHolder {
+/// \ingroup ControlSystemGroup
+/// Holds all options for a single control system
+///
+/// This struct collects all the options for a given control system during
+/// option parsing. Then during initialization, the options can be retreived as
+/// a TaggedTuple and assigned to their corresponding DataBox tags.
+template <size_t DerivOrder>
+struct ControlSystem {
```

Comment:
This class would be simpler as a "C-style" struct with public member variables.  It wouldn't need any member functions except `pup`.

Reply:
Ok I'll just expose the member variables. Then I don't have to bother with the `TaggedTuple`s and such.